src/Orchard/Environment/WorkContextModule.cs

```diff
@@ -23,6 +23,10 @@ public class WorkContextModule : Module {
             builder.RegisterType<WorkContextProperty<HttpContextBase>>()
                 .As<WorkContextProperty<HttpContextBase>>()
                 .InstancePerMatchingLifetimeScope("work");
+                

```

Comment:
This change should be removed as it is the opposite as the lines before. An httprequest object has to be per work scope.